docs/enhancement-proposals/off-ep-0010.md

```diff
@@ -0,0 +1,140 @@
+
+In addition, a SMIRKS pattern that can match a particular 
+bonded quartet in either `i-j-k-l` or `l-k-j-i` order is 
+ambiguous, and the SMIRNOFF specification does not 
+guarantee that the match will be performed in any predetermined or deterministic order. As this may
+potentially lead to undesired results, this proposal
+adds a note highlighting this fact.
+
+Finally, the effect of the `idivf="auto"` parameter on the ProperTorsion potential is outlined in words that can be interpreted ambiguously.
+
+All of these ambiguities may cause confusion to anybody implementing the SMIRNOFF spec.
+
+## Usage and Impact
+
+This change defines the ``idivf`` parameter in line with the philosophy used in AMBER force fields. The OpenFF implementation has not hitherto used or implemented ``idivf="auto"`` in its ProperTorsion parameters. As such there should be no practical impact on existing workflows and simulations. Other force fields and software that have implemented and interpreted the `idivf="auto"` parameter in ways that do not align with our definition will need to be updated accordingly.
+
+The proposed clarification of input vector order and symmetric SMIRKS matching reflects existing implementations and simulations in OpenFF and OpenMM. Software that converts systems out into other formats may need to adjust the input order to ensure the correct sign of the torsion.
+
+## Backward compatibility
+
+This proposal does not change behaviour, but rather explicitly defines what is currently implicit in the
+specification and the implementation in OpenFF infrastructure.
+Therefore, there should be no backwards compatibility issues.
+
+## Detailed description
+
+This proposal adds the following section (bolded) clarifying ``idivf`` to the ``ProperTorsion`` spec:
+
+> For convenience, an optional attribute specifies a torsion multiplicity by which the barrier height (``k#``) should be divided (`idivf#`). **The final barrier height is calculated as ``k#/idivf#``. ``idivf`` can be assigned an integer value (such as `"1"`), or `"auto"`. If `idivf="auto"`, the following equation is used to determine the ``idivf`` value for a torsion applying to four atoms `i-j-k-l`, where ``n_j`` refers to the degree (i.e. number of bonds) of atom `j`:**
+> 
+> **```**
+> **idivf = (n_j - 1) * (n_k - 1)**
+> **```**
+> 
+> The default behavior of this ``idivf`` can be controlled by the top-level attribute `default_idivf` (default: `"auto"`) for `<ProperTorsions>`.
+
+It also adds a section explaining the computation of ``theta`` to the ``ProperTorsion`` spec:
+
+> In the potential function, the angle ``theta`` is calculated using input vectors
+> defined by the four atoms of the torsion `i-j-k-l`.
+> Where the vector ``r_ij`` is defined as the vector from atom `j` to atom `i`:
+> ```
+> r_ij = x_i - x_j
```

Comment:
This definition is backwards, isn't it? This vector points from j to i

Reply:
Yes -- I chose to call it `r_ij` instead of `r_ji` for easier comparison to OpenMM's implementation, which also names vectors like that, but happy to switch to `r_ji` instead if that reads better.

Reply:
Which OpenMM docs are you referring to? This is not in the [OpenMM User Guide](http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html#periodictorsionforce).

Reply:
Let's use standard vector notation `r_ij = x_j - x_i`